src/App.js

```diff
@@ -5,6 +5,8 @@ import Home from './components/homepage/Home';
 import Profile from './components/profile/Profile';
```

Comment:
we have now 'pages' folder so adjust your code to 
import Profile from './components/pages/profile/Profile'
so git will let me approve it